draft-ietf-rats-eat.md

```diff
@@ -2670,3 +2670,9 @@ differences. A comprehensive history is available via the IETF Datatracker's rec
 - Several edits and clarifications for freshness and nonces
 - Correct eat_nonce registration for JSON-encoded tokens
 - Add security considerations for freshness
+
+Contributors
```

Comment:
Do you just want names here (as in Acknowledgements) or an actual contributors section (with identifying information etc.)?

Reply:
Contributors are derived from https://github.com/ietf-rats-wg/eat/graphs/contributors.  Acknowledgments would be a different section.

Reply:
OK, I'll make a PR